src/components/RepoList.js

```diff
@@ -1,23 +1,25 @@
 import React, {PropTypes} from 'react';
 import RepoPreview from './RepoPreview';
 import {Route} from 'react-router-dom';
+import List, { ListItem, ListItemText } from 'material-ui/List';
+import Divider from 'material-ui/Divider';
 
 const RepoDetail = () => {
   return <span>Show Repo Information Here</span>;
 }
 
 const RepoList = (props) => (
   <div>
-    <ul>
+    <List>
       {props.repos.map((item, index) => (
-        <li>
-        <RepoPreview
-          id={item.id}
-          title={item.title}
-         />
-        </li>
+        <ListItem button>
+          <RepoPreview
```

Comment:
If you want to pass every property of `item` to your component, a handy shorthand syntax would be: `<RepoPreview {...item} />`

Reply:
That is handy, do you know if there is a way  that id can be passed as a number instead of a string using the spread operator that way? I like the short hand version, but I get an error from my Repo Preview component stating that id from item was given as a string, when it expected a number from the prop types.